### ISSUE TITLE
Would this be compatible with Django 3?

### ISSUE THREAD

**Issue by nh916** [stephenmcd/django-socketio#120]
Hi I am wondering if this would be compatible with Django 3?

**Comment by guettli** [comment]
@stephenmcd  Good question. This projects does not look healthy. Please tell us the truth about the state of the project. It is 100% ok if you have no time. Please talk about this, this helps to find a simple solution.

**Comment by pupattan** [comment]
I have the same question also

**Comment by Marco-Gtto** [comment]
The real question is: why was this project abandoned?

 @stephenmcd  There are a lot of people who would like this repository, not everyone has the time to implement socket.io in django, especially if it's a small project.
bring this repository back to life!

**Comment by nh916** [comment]
also django channels sucks and nodejs socketio is good. if we can get the same socket io in django that would be gold!

**Comment by avidcoder123** [comment]
It's sad to see this repository go; this seems like the perfect websocket implementation out there since Django channels is confusing and hard to use. However, I found an alternative called Python-EngineIO. It's not as good as this repo, but it's the easiest solution out there.

**Comment by nh916** [comment]
@thomcatmeow can we maybe somehow maintain it and bring in more developers? if we can contribute we can keep it and all and more can use it

**Comment by avidcoder123** [comment]
@nh916 Yes that would be even better than finding an alternative. With more developers, we can keep this amazing repository working for everyone.

**Comment by iamtekson** [comment]
I think it is better to move to [django channel](https://github.com/django/channels).
